src/sentry/integrations/pagerduty/client.py

```diff
@@ -64,6 +64,8 @@ def send_trigger(
                     "component": group.project.slug,
                     "custom_details": custom_details,
                 },
+                "client": "sentry",
+                "client_url": group.get_absolute_url(params=link_params),
                 "links": [
                     {
                         "href": group.get_absolute_url(params=link_params),
```

Comment:
minor nit: put `group.get_absolute_url(params=link_params)` in a variable since we use it at least twice to save some computation

Reply:
Hi, I refactored. Is there anything I can do to get this PR merged?

Reply:
@ykamo001 @iamrajjoshi @schew2381 
Hi, this small change could be very helpful to my organization which is in Sentry Business 
 plan and comparing features with datadog. Could you please help merge this PR?

And may I ask what's the timeline to production?

Reply:
Hi sorry this slipped through, there's just one failing test that I left a fix for

Reply:
Once that change goes through, I'll run the tests again and then merge it!